### PR TITLE
Add routing for art and games

### PR DESCRIPTION
**Pull request by zapplecat**

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <BrowserRouter>
     <App />
-  </React.StrictMode>,
+  </BrowserRouter>,
```
Comment (Nephren68) [inline frontend/src/index.tsx:11]: Why remove `StrictMode`?

Comment (zapplecat) [reply frontend/src/index.tsx:11]: Got a "Error: Invariant failed: You should not use <Switch> outside a <Router>". But we can put the StrictMode around the router like
```
  <React.StrictMode>
      <BrowserRouter>
        <App />
      </BrowserRouter>
  </React.StrictMode>,
```
Anything wrong with this approach?

Comment (zapplecat) [reply frontend/src/index.tsx]: Looking at `StrictMode` docs closer, looks like it just applies everything to decadents. Added it back in.

**Review by ProgrammaticNajel — APPROVED** [review]

**Review by Nephren68 — APPROVED** [review]

**Comment by zapplecat** [comment]
[add routing for gallery and games](https://trello.com/c/6shOvWnH/77-add-routing-for-gallery-and-games)

**Pull request merged**